extensions/servlet/test/com/google/inject/servlet/ServletDefinitionPathsTest.java

```diff
@@ -216,35 +229,39 @@ protected void service(HttpServletRequest servletRequest,
 
   // Data-driven test.
   public final void testPathInfoWithRegexMatching() throws IOException, ServletException {
+
     // first a mapping of /*
-    pathInfoWithRegexMatching("/path/index.html", "/path", "/(.)*", "/index.html", "");
-    pathInfoWithRegexMatching("/path//hulaboo///index.html", "/path", "/(.)*",
+    pathInfoWithRegexMatching("/path/index.html", "/path", "(/).*", "/index.html", "");
```

Comment:
IIUC, there's no change between these two regexs -- the former put a group around the "match any character" so the * was for the group.  the latter puts the group around the "/".  why did you change this?